web/app/scripts/views/record/add-edit-partial.html

```diff
@@ -42,7 +42,7 @@
                             </div>
                         </div>
                         <div class="col-md-12">
-                            <div class="map" leaflet-map driver-embed-map style="height:300px;"></div>
+                            <div class="map" leaflet-map driver-embed-map editable="true"></div>
```

Comment:
If you have the initial coordinates before the map loads, that might be a great idea.  But I imagine there might be a timing issue with the record and map both loading asynchronously, and the record coordinates not necessarily populating before the map gets set up.

For the editable map, it will still need the events. Because there are multiple directives involved, I don't think this directive can go put/read things on the scope.